### PR TITLE
add header tag only

### LINKED ISSUE #2: add header
The header should be responsive and include a spot for the logo on the left side and the nav tag on the right side. Just add home and contact us to the nav bar for now. Use # for the links.

### PR DESCRIPTION
**Pull request by Gjhnsn**
wasnt sure of exactly what you wanted with nav so left it empty

fixes #2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,8 +13,9 @@
     <title>Hello, world!</title>
   </head>
   <body>
+    <header></header>
```
Comment (vanderpool23) [inline index.html:16]: Good start.

**Review by vanderpool23 — CHANGES_REQUESTED** [review]
Let's meet and talk about the changes for the nav bar.

File: index.html (hunk 1 above)
Comment (Gjhnsn) [reply index.html]: @vanderpool23 thanks

Comment (vanderpool23) [reply index.html]: @Gjhnsn don't get cocky kid

**Review by vanderpool23 — APPROVED** [review]

**Pull request merged**